prometheus/traefik-pass

```diff
@@ -0,0 +1 @@
+change-this-password
```

Comment:
Adding this file is a mistake, right? It is not used anymore since 3659e1a1d98280f7f15fa22787e8ff170150f856 (and was never version controlled)